calculator.sh

```
@@ -6,7 +6,7 @@ echo "Enter first number:"
 read num1
 
 echo "Enter second number:"
-read num4
+read num10
```

Shellcheck (0.10.0)

[warning] num10 appears unused. Verify use (or export if used externally).

(SC2034)